src/app/navigationManager.ts

```diff
@@ -45,6 +45,12 @@ export class NavigationManager {
 
         const parsedURL = parseURL(url)!;
         const server = ServerManager.lookupServerByURL(parsedURL, true);
+
+        let trimmedPathname = getFormattedPathName(parsedURL.pathname);
```

Comment:
I don't think this is necessary, `getFormattedPathName` basically just adds a `/` to the end of the pathname:

```
export const getFormattedPathName = (pn: string) => (pn.endsWith('/') ? pn : `${pn}/`);
```

So if we want to remove it, we should just use `parsedURL.pathname`